src/main/java/transforms/GeocoderTransform.java

```diff
@@ -50,6 +67,10 @@ public GeocoderTransform(
     }
 
     public String getAddress( String input, String addressObject ) {
+        if ( !Arrays.asList(LAT, LON, GEOGRAPHY_POINT, TYPE, HOUSE_NUMBER,
+                ROAD, NEIGHBORHOOD, CITY, POSTCODE, COUNTY, STATE).contains(this.addressObject) ) {
```

Comment:
This operation will be faster if you use a HashSet instead of a List (constant time lookup) -- and even faster if you make the set a static variable up top so you don't have to reconstruct it every time (
```
private static final Set<String> addressObjects = Sets.newHashSet( LAT, LON, ... );
```

Reply:
Ooh ok. I was thinking there was only one lookup so HashSet wasn't worth it (construction is O(n)). But static fixes that.